apps/dcos_overlay/src/dcos_overlay_lashup_kv_listener.erl

```diff
@@ -132,7 +164,27 @@ to_str({IP, Prefix}) ->
 to_str(Mac) when length(Mac) =:= 6 ->
     List = [[integer_to_list(A div 16, 16),
              integer_to_list(A rem 16, 16)]
-           || A <- Mac],
+            || A <- Mac],
     lists:flatten(string:join(List, ":"));
 to_str(IP) ->
     inet:ntoa(IP).
+
+%%%===================================================================
+%%% Metrics functions
+%%%===================================================================
+
+-spec(init_metrics() -> ok).
+init_metrics() ->
+    prometheus_counter:new([
+        {registry, overlay},
+        {name, updates_total},
```

Comment:
there is `update_processing_duration_seconds_cnt`

Reply:
Hmm. There are updates from Lashup, which are accounted for by this metric, and reconciliations too, and for all of them `update_processing_duration_seconds` is used. Now I am thinking of a better word instead of `update`